### DISCUSSION TITLE
Audio from camera microphone

### DISCUSSION THREAD

**Discussion by Disincentivato** [fuatakgun/eufy_security#1036] [Q&A]
Is there a way to also have the camera audio?
At the moment I receive the video signal via RTSP, I have the volume control but I don't receive any audio from the cameras.

**Comment by Disincentivato** [comment]
Audio? No audio? si audio? :)

**Comment by bjaurelio** [comment]
I don't have any audio either and came here looking to see if that was possible. It appears not.
